features/session-replay/src/iosMain/kotlin/com/datadog/kmp/sessionreplay/UIViewExt.kt

```diff
@@ -0,0 +1,93 @@
+import cocoapods.DatadogSessionReplay.DDImagePrivacyLevelOverrideMaskNone
+import cocoapods.DatadogSessionReplay.DDImagePrivacyLevelOverrideNone
+import cocoapods.DatadogSessionReplay.DDTextAndInputPrivacyLevelOverride
+import cocoapods.DatadogSessionReplay.DDTextAndInputPrivacyLevelOverrideMaskAll
+import cocoapods.DatadogSessionReplay.DDTextAndInputPrivacyLevelOverrideMaskAllInputs
+import cocoapods.DatadogSessionReplay.DDTextAndInputPrivacyLevelOverrideMaskSensitiveInputs
+import cocoapods.DatadogSessionReplay.DDTextAndInputPrivacyLevelOverrideNone
+import cocoapods.DatadogSessionReplay.DDTouchPrivacyLevelOverride
+import cocoapods.DatadogSessionReplay.DDTouchPrivacyLevelOverrideHide
+import cocoapods.DatadogSessionReplay.DDTouchPrivacyLevelOverrideNone
+import cocoapods.DatadogSessionReplay.DDTouchPrivacyLevelOverrideShow
+import cocoapods.DatadogSessionReplay.ddSessionReplayPrivacyOverrides
+import com.datadog.kmp.sessionreplay.configuration.ImagePrivacy
+import com.datadog.kmp.sessionreplay.configuration.TextAndInputPrivacy
+import com.datadog.kmp.sessionreplay.configuration.TouchPrivacy
+import platform.Foundation.NSNumber
+import platform.Foundation.numberWithBool
+import platform.UIKit.UIView
+
+/**
+ * Allows setting a view to be "hidden" in the hierarchy in Session Replay.
+ * When hidden the view will be replaced with a placeholder in the replay and
+ * no attempt will be made to record it's children.
+ *
+ * @param hide pass `true` to hide the view, or `false` to remove the override
+ */
+fun UIView.setSessionReplayHidden(hide: Boolean) {
+    ddSessionReplayPrivacyOverrides().setHide(NSNumber.numberWithBool(hide))
+}
+
+/**
+ * Allows overriding the image privacy for a view in Session Replay.
+ *
+ * @param privacy the new privacy level to use for the view
+ * or null to remove the override.
+ */
+fun UIView.setSessionReplayImagePrivacy(privacy: ImagePrivacy?) {
+    ddSessionReplayPrivacyOverrides().setImagePrivacy(privacy.nativeOverride)
+}
+
+/**
+ * Allows overriding the text and input privacy for a view in Session Replay.
+ *
+ * @param privacy the new privacy level to use for the view
+ * or null to remove the override.
+ */
+fun UIView.setSessionReplayTextAndInputPrivacy(privacy: TextAndInputPrivacy?) {
+    ddSessionReplayPrivacyOverrides().setTextAndInputPrivacy(privacy.nativeOverride)
+}
+
+/**
+ * Allows overriding the touch privacy for a view in Session Replay.
+ *
+ * @param privacy the new privacy level to use for the view
+ * or null to remove the override.
+ */
+fun UIView.setSessionReplayTouchPrivacy(privacy: TouchPrivacy?) {
+    ddSessionReplayPrivacyOverrides().setTouchPrivacy(privacy.nativeOverride)
+}
+
+private val ImagePrivacy?.nativeOverride: DDImagePrivacyLevelOverride
+    get() = when (this) {
+        ImagePrivacy.MASK_NONE -> DDImagePrivacyLevelOverrideMaskNone
+        ImagePrivacy.MASK_LARGE_ONLY -> DDImagePrivacyLevelOverrideMaskNonBundledOnly
```

Comment:
The need to be the same, because they are coming from the `common` declaration. I explained the difference in the behaviour between the platforms here https://github.com/DataDog/dd-sdk-kotlin-multiplatform/pull/112/files#diff-132547c2c660701c9fb15ee9845e4c46066d25595aee1ede06cc2f445cdee294R21-R25